content/blog/2019/11/2019-11-17-gsoc-report.adoc

```diff
@@ -0,0 +1,224 @@
+---
+layout: post
+title: "Google Summer of Code 2019 Report"
+tags:
+- gsoc
+- gsoc2019
+- jenkins
+- community
+- events
+authors:
+- martinda
+- jeffpearce
+- oleg_nenashev
+- markyjackson-taulia
+---
+
+Google Summer of Code (GSoC) 2019 is officially completed and we are happy to announce that five projects were successfully completed.
+We held the final presentations in late August and Google published the results on Sept 3rd.
+
+We would like to thank all our students, their mentors and all other contributors who proposed project ideas, participated in student selection, in community bonding and in further reviews.
+Google Summer of Code is a major effort which would not be possible without the active participation of the Jenkins community.
+
+The final presentations can be found here:
+
+* link:https://www.youtube.com/watch?v=g19o24uzy6c[Part 1 presentations]
+* link:https://www.youtube.com/watch?v=cMSbdGwGWp0[Part 2 presentations]
+* link:https://www.youtube.com/watch?v=41dhyFC5Iak[Part 3 presentations]
+
+== Summary
+
+
+This year GSoC was a year-round effort for the organization and some community members.
+I would like to start this report by thanking all participants, students, mentors, subject matter experts and community members who support the project and the students.
+We could not do this without you.
+We presented the
+link:https://docs.google.com/presentation/d/e/2PACX-1vQf5RLIJXq8cPEKbCIaA4Bn3tMy5ITB4z2oGJKwLOJz_ne2AiNuSSvXi51nMDDDgA/pub?start=false&loop=false&delayms=3000[2019 Jenkins GSoC report]
+at link:https://www.cloudbees.com/devops-world/san-francisco[DevOps World | Jenkins World San Francisco] conference.
```

Comment:
TODO; Lisbon

Reply:
I added a link to the Lisbon conference page.